azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_stages_provisioning.py

```diff
@@ -44,3 +58,327 @@ def _execute_op(self, op):
 
         else:
             self.send_op_down(op)
+
+
+class CommonProvisioningStage(PipelineStage):
+    @pipeline_thread.runs_on_pipeline_thread
+    def _get_error(self, provisioning_op, prov_op_name, error):
+        if error:
+            return error
+        elif 300 <= provisioning_op.status_code < 429:
```

Comment:
>provisioning_op.status_code < 429: [](start = 20, length = 34)

what about status codes 430?  It looks like those all get converted into success.  Is that what you want to do?

Reply:
may be we need to speak. This was the exact behavior in polling state machine. Anything >= 429 was going to retry the process again in the PUT Register as well as GET Query

---
In reply to: [350960869](https://github.com/Azure/azure-iot-sdk-python/pull/391#discussion_r350960869) [](ancestors = 350960869)